src/applications/mhv-medical-records/containers/LandingPage.jsx

```diff
@@ -369,20 +388,15 @@ const LandingPage = () => {
               Where can I find health information I entered myself?
             </h3>
             <p className="vads-u-margin-bottom--2">
-              Right now, your records on VA.gov only include health information
-              your VA providers have entered.
-            </p>
-            <p className="vads-u-margin-bottom--2">
-              To find health information you entered yourself, go to your
-              medical records on the My HealtheVet website.
+              Download your self-entered health information report.
             </p>
             <p className="vads-u-margin-bottom--2">
               <ExternalLink
                 href={mhvUrl(
                   isAuthenticatedWithSSOe(fullState),
                   'download-my-data',
                 )}
-                text="Go to your medical records on the My HealtheVet website"
+                text="Go to download your medical records reports"
```

Comment:
Like the changes to "Download your medical records reports" above, these changes should be behind the `phase0p5Flag` feature flag.

- If the flag is false, keep the existing content and link.
- If the flag is true, update to the new content and update the link to the "/download" route